Clarify docs for unifi network sensors (#28476)
home-assistant/home-assistant.github.io: source/_integrations/unifi.markdown

## Sensor

> These Sensor are disabled by default, to enable them, click the "Configure" button on the Unifi-integration page and go to page 3/3. Here the different sensors can be enabled.

Reviewer: ```suggestion
These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.

1. To enable the sensors, on the Unifi integration page, select **Configure**. 
2. Go to page 3/3 and enable the sensors.
```

> **Warning:** This will add a new Device to Home Assistant for every connected client in your network. If you network is very large, this will get very messy in Home Assistant.

Reviewer: ```suggestion
```
Moved this information up. I think it is worth mentioning this before showing how to enable the sensors.

> Currently it is also not possible to get a global bandwidth sensor for a UDM router. The Integration only supports per-device sensors.

Reviewer: ```suggestion
Currently, it is not possible to get a global bandwidth sensor for a UDM router. The integration only supports per-device sensors.
```

## Sensor

These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.

> 1. To enable the sensors, on the Unifi integration page, select **Configure**. 

Reviewer: ```suggestion
1. To enable the sensors, on the UniFi integration page, select **Configure**. 
```

2. Go to page 3/3 and enable the sensors.

> Currently, it is not possible to get a global bandwidth sensor for a UDM router. The integration only supports per-device sensors.

Reviewer 2: I would remove this comment as we typically don't list w Har we don't do with an integration

Reviewer: I understand. 
@Kane610, would you be ok with keeping the more generic phrase _The integration only supports per-device sensors_ ?

Author: You're right, usually this isn't mentioned but in this case there were a lot of questions about this exact scenario (eg. [this issue](https://github.com/home-assistant/home-assistant.io/issues/28463#issuecomment-1666951580)) and some more threads in the community. Therefore, I thought it might be a good idea to include this here.
However, if this shouldn't be mentioned here, we can of couse remove it. 😉

@c0ffeeca7 This is also an option. 👍

Reviewer 2: A discussion between two people hardly qualify the need for clarifying it as it's been available for two years :)

Yes the text could be reformulated that the sensors are only for individual network clients

Author: Well, you're probably right ;-)
I'll change that to *The integration only supports per-device sensors.* 

**Just a question:** Do I need to open a new PR or is there another way of merging these additional changes?

Reviewer: Yes, it requires a separate PR. I can do it, since I'm the one who's guilty of merging this without waiting for input from the owner.

Author: That's great, thank you! 👍

Reviewer: #28490

## Sensor

These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.

1. To enable the sensors, on the Unifi integration page, select **Configure**. 
2. Go to page 3/3 and enable the sensors.

> 

Reviewer: ```suggestion
```
